gax-httpjson/src/main/java/com/google/api/gax/httpjson/ApiMessageHttpRequestFormatter.java

```diff
@@ -62,37 +60,14 @@
   @Override
   public abstract PathTemplate getPathTemplate();
 
-  protected abstract Gson getRequestMarshaller();
-
   private static <RequestT extends ApiMessage> ApiMessageHttpRequestFormatter<RequestT> create(
-      final RequestT requestInstance,
       Set<String> queryParams,
       String resourceNameField,
       ResourceNameFactory resourceNameFactory,
       PathTemplate pathTemplate) {
 
-    final Gson baseGson = new GsonBuilder().create();
```

Comment:
This whole block turns out to be unnecessary; the default Gson will know how to serialize a straightforward POJO.